test/javascripts/turbolinks_test.coffee

```diff
@@ -1,210 +1,946 @@
 describe 'Turbolinks', ->
-
-  createTurboNodule = ->
-    $nodule = $("<div>").attr("id", "turbo-area").attr("refresh", "turbo-area")
-    return $nodule[0]
-
-  url_for = (slug) ->
+  server = null
```

Comment:
600 lines to the first test!  Can the test data be loaded dynamically?  Ideally, I'd be able to look at the tests in one editor pane, and open relevant test data in another.